### ISSUE TITLE
alert positioning should be available for ipad 

### ISSUE THREAD

**Issue by soumitra12** [huri000/SwiftEntryKit#218]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by soumitra12** [comment]
The offset parameter sets the offset from both sides, which is fine in case of iphone
for iPad there should be a parameter to set it from one side instead of both sides

**Comment by huri000** [comment]
It's a bit unclear what you are trying to do. Could you please be more accurate by describing exactly the feature you wish implemented, and by providing a screenshot of the current state with further explanations on the things you want to be changed/added.

**Comment by soumitra12** [comment]
// Give the entry the width of the screen minus 20pts from each side, the height is decided by the content's contraint's
attributes.positionConstraints.size = .init(width: .offset(value: 20), height: .intrinsic)
I want screen minus 20pts from one side
Since i am using ipad i want to show the alert at screenwidth/2 position

**Comment by huri000** [comment]
Makes sense but I don't think it's an iPad specific feature. I'll keep you posted on this one.

**Comment by soumitra12** [comment]
yeah you are right this is not iPad specific issue, It can be a requirement in iphone as well
I have featured dependent completely on this change 
I request you to please provide the same to the earliest
